.github/workflows/update_sdk_methods.py

```diff
@@ -79,6 +79,35 @@
 ## Build path to sdk_protos_map.csv file that contains proto-to-methods mapping, used in write_markdown():
 proto_map_file = os.path.join(gitroot, '.github/workflows/sdk_protos_map.csv')
 
+## Check to see if we have a locally-staged version of the Go SDK Docs (RDK repo). If so,
+## scrape our code samples (and only code samples!) from that URL instead. This check just
+## establishes whether the URL is up or not; if detected as up here, it is scraped in parse().
+## TODO: Consider if we need to consider other ports besides '8080', i.e. if multiple stage attempts,
```

Comment:
Great comment, thank you!! Mostly, it would be we technical writers running this script, so if you encountered the default port being in use already, I should definitely add more robust checking. Thanks for reporting, I'll add this now!

Reply:
Now updated to be able to handle whichever port `pkgsite` is listening on! Thanks for the report!